### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by cbalster**
Just some minor updates
@ligi I just stole the first commit from your current PR

### CODE REVIEW

File: app/src/main/res/layout/fragment_chat.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
                         android:layout_height="wrap_content"
                         android:hint="@string/chat_text"
                         android:imeOptions="actionSend"
-                        android:singleLine="true"/>
+                        android:maxLines="1"/>
```
Comment (ligi) [inline app/src/main/res/layout/fragment_chat.xml]: can you explain this change? *curious*

Comment (cbalster) [inline app/src/main/res/layout/fragment_chat.xml]: lint says singleLine is deprecated ;)

**Comment by ligi** [comment]
LGTM - now you face the same travis fuckup I had with my PR ..
perhaps let's update to latest okhttp ( 3.x ) when we update

File: app/src/main/res/layout/fragment_chat.xml (hunk 1 above)
Comment (ligi) [inline app/src/main/res/layout/fragment_chat.xml]: makes sense

**Comment by ligi** [comment]
good job with figuring out the travis problem @cbalster !

**Pull request merged**